### DISCUSSION TITLE
Installation Error - PHP Error - QRZ Sent

### DISCUSSION THREAD

**Discussion by itsthemedication** [magicbug/Cloudlog#2842] [Q&A]
OS: Linux Mint 21 
Cloudlog 2.6.0
Server Software	Apache/2.4.52 (Ubuntu)
PHP Version	8.1.2-1ubuntu2.14
MySQL Version	8.0.35-0ubuntu0.22.04.1
-------------------------------------------------------------
So I plowed my way through the wiki install and was able to login and I get the dashboard but I get four PHP errors at the top of the page:

A PHP Error was encountered
Severity: Warning
Message: Undefined array key "QRZ_Sent"
Filename: controllers/Dashboard.php
Line Number: 110

Other three are exactly the same just line numbers "QRZ_Received" line 111, "QRZ_Sent_today" line 112, and "QRZ_Received_today" line 113.

I installed the qrz user/passwd to application/config/config.php and aslo installed the api in the dashboard.
Under maintenance and debug everything shows success.
Thanks in advance.

**Comment by phl0** [comment]
What does the migration number in debug info say?

**Reply by itsthemedication** [reply]
160

**Comment by phl0** [comment]
Fix proposed in https://github.com/magicbug/Cloudlog/pull/2844.

**Reply by magicbug** [reply]
I've merged this into the master branch too
